packages/position-managers/src/constants/vaults/bsc.ts

```diff
@@ -4,6 +4,28 @@ import { Strategy, VaultConfig } from '../../types'
 import { MANAGER } from '../managers'
 
 export const vaults: VaultConfig[] = [
+  {
+    id: 15,
+    idByManager: 1,
+    name: 'ALPACA',
+    bCakeWrapper: '0x98385EBAC7055a569b76C66Dd559055263Bc88da',
+    address: '0x98385EBAC7055a569b76C66Dd559055263Bc88da',
+    adapterAddress: '0x778E55EeAA3cC7F590272aC4E173d43012271bad',
+    currencyA: bscTokens.eth,
+    currencyB: bscTokens.btcb,
+    earningToken: bscTokens.cake,
+    feeTier: FeeAmount.MEDIUM,
+    strategy: Strategy.SAVINGS,
+    manager: MANAGER.ALPACA,
+    isSingleDepositToken: true,
+    allowDepositToken0: false,
+    allowDepositToken1: true,
+    managerInfoUrl: 'https://www.alpacafinance.org/',
+    strategyInfoUrl: 'https://docs.alpacafinance.org/leveraged-yield-farming/strategies',
+    learnMoreAboutUrl: 'https://docs.alpacafinance.org/leveraged-yield-farming/strategies/pancakeswap-farms',
+    priceFromV3FarmPid: 8,
+    minDepositUSD: 50,
```

Comment:
Anyway, to confirm `minDepositUSD` amount?

Reply:
yes, for this vault, there's a min deposit usd limit

Reply:
I mean this `50`, can I find in which contract?

Reply:
let me check

Reply:
![CleanShot 2024-02-01 at 16 19 47@2x](https://github.com/pancakeswap/pancake-frontend/assets/99634186/d44519cd-046f-487c-9dc1-c77992d854c0)
need to read from another contract 
will add a test case for it